### DISCUSSION TITLE
Safe to use with 7735HS?

### DISCUSSION THREAD

**Discussion by mja87** [DavidS95/Smokeless_UMAF#18] [Q&A]
I got a new MiniPC who's iGPU would benefit from the faster RAM I have, but the Ryzen 7 7735HS is locked at 4800 MHz. 

Is this one of the "dangerous devices" that should not use this tool, or am I okay to try to over clock the ram?

**Comment by l33tm4st3r** [comment]
Just do a bios dump backup, before test any option.

Buy a ch341a programer with a 1.8v converter if you need to unbrick your new toy.  

https://www.amazon.com/KOOBOOK-CH341A-Programmer-Adapter-Converter/dp/B07VMPZFWH/ref=sr_1_4?__mk_es_US=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=UYPVIJP65YZ0&currency=USD&keywords=ch341a&qid=1680105953&sprefix=ch341a%2Caps%2C186&sr=8-4

**Comment by DavidS95** [comment]
Try the new beta version I uploaded it should work on the newest gen better.
